test/cypress/integration/settings/your-psp.cy.js

```diff
@@ -351,6 +326,18 @@ describe('Your PSP settings page', () => {
     })
 
     it('should allow MOTO merchant account code', () => {
+      cy.task('setupStubs', [
+        ...getUserAndGatewayAccountStubs(gatewayAccountOpts),
+        gatewayAccountStubs.postCheckWorldpayCredentials({ ...testCredentialsMOTO, gatewayAccountId }),
+        gatewayAccountStubs.patchUpdateWorldpayOneOffCredentialsSuccess({
+          gatewayAccountId,
+          credentialId: credentialsId,
+          merchantCode: testCredentialsMOTO.merchant_id,
```

Comment:
Do we want this yet? Aren’t we still patching using the legacy structure with `merchant_id`?